updated readme to include CICD workflow (#2493)
w3f/polkadot-wiki: README.md

The Polkadot wiki is built on the `gh-pages` branch and automatically deployed to GitHub Pages.
The Kusama wiki is also deployed to GitHub Pages (via a separate repository).

Development servers exist at `https://staging.polkadot.network` and `https://staging.kusama.network`.
The servers will reflect the latest `master` commit or PR put up against the master branch by a member of the Technical Education team.

> When the latest `master` is ready to be pushed to production, merge/rebase `master` into `prod` and push. CICD will deploy `prod` to

Reviewer: Wouldn't this be for just the team?

> When the latest `master` is ready to be pushed to production, merge/rebase `master` into `prod` and push. CICD will deploy `prod` to

Reviewer: ```suggestion
The latest version of `master` is staged and checked by the team. If all is well, the new commits on `master` are transferred into the production branch,`prod`, by rebasing `master` on `prod`. The CICD production workflow will deploy `prod` to the public sites:
```

> `https://wiki.polkadot.network` and `https://guide.kusama.network`, respectively.

Reviewer: ```suggestion
[Polkadot Wiki](https://wiki.polkadot.network) and [Kusama Guide](https://guide.kusama.network), respectively.
```

> When merging `master` to `prod`, you should not see anything other than a [fast-forward](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging).

Reviewer: ```suggestion
```

> If it gives you an error or generates a merge commit, you probably have a commit directly on `prod` that should not be there.

Reviewer: ```suggestion
```
